packages/components/src/components/multi-select/multi-select/multi-select-styles.ts

```diff
@@ -0,0 +1,204 @@
+import type { BreakpointCustomizable, Theme } from '../../../types';
+import { buildResponsiveStyles, getCss, SelectDropdownDirectionInternal } from '../../../utils';
+import {
+  addImportantToEachRule,
+  addImportantToRule,
+  getHiddenTextJssStyle,
+  getThemedColors,
+  getTransition,
+  hostHiddenStyles,
+  hoverMediaQuery,
+} from '../../../styles';
+import {
+  borderRadiusSmall,
+  borderWidthBase,
+  fontLineHeight,
+  spacingStaticMedium,
+  spacingStaticSmall,
+  spacingStaticXSmall,
+  textSmallStyle,
+} from '@porsche-design-system/utilities-v2';
+import type { Styles } from 'jss';
+import { getThemedFormStateColors } from '../../../styles/form-state-color-styles';
+import type { FormState } from '../../../utils/form/form-state';
+import { getLabelStyles } from '../../../styles/form-styles';
+import { getPlaceholderJssStyle } from '../../../styles/placeholder';
+import { getNoResultsOptionJssStyle, MULTI_SELECT_OPTION_HEIGHT } from '../../../styles/select';
+import { getFunctionalComponentRequiredStyles } from '../../common/required/required-styles';
+import { getFunctionalComponentStateMessageStyles } from '../../common/state-message/state-message-styles';
+
+const inputYPadding = '13px';
```

Comment:
typo?
```suggestion
const inputPadding = '13px';
```

Reply:
It is used as Y padding thus the name :)